public/app/features/explore/Wrapper.test.tsx

```diff
@@ -281,10 +283,7 @@ async function changeDatasource(name: string) {
   const datasourcePicker = await screen.findByLabelText(selectors.components.DataSourcePicker.container);
   // Bit awkward here but we need to fire the event on some child component that we don't have any label for.
   // We do not have label because we do not simulate proper element width and so we are using small styles where
-  // label is missing.
-  screen.debug(datasourcePicker.children[0].children[0].children[0], 20000);
-
-  // The select uses mouseDown for opening
+  // label is missing. Also the select uses mouseDown for opening.
   fireEvent.mouseDown(datasourcePicker.children[0].children[0]);
```

Comment:
Just a nit, can we do something like:
```ts
const selectElement = datasourcePicker.children[0].children[0];
fireEvent.mouseDown(selectElement);
```
(not sure if `selectElement` is the proper name) to make what we are clicking on a little bit more explicit?

Reply:
sure, though to be honest not exactly what are we clicking on there. The select element is fairly custom so it's probably divs all the way down.